OrbitLinuxTracing/PerfEventOpen.h

```diff
@@ -24,14 +25,14 @@ namespace LinuxTracing {
 inline void perf_event_reset(int file_descriptor) {
   int ret = ioctl(file_descriptor, PERF_EVENT_IOC_RESET, 0);
   if (ret != 0) {
-    ERROR("PERF_EVENT_IOC_RESET: %d", ret);
+    ERROR("PERF_EVENT_IOC_RESET: %s", strerror(errno));
```

Comment:
Is there a case where this can actually be a problem other than our code modifying the string returned by strerr or calling strerror_l?
In case we want to do it anyway, do you have suggestions for the file in OrbitBase in which to put it?

Reply:
> In case we want to do it anyway, do you have suggestions for the file in OrbitBase in which to put it?

Still Logging.h?

Reply:
It is problematic, because implementation reuses the same buffer and the buffer is not thread_local.

I think it should go to separate header - SafeStrerror.h?

Reply:
Alright, have a look.